### DISCUSSION TITLE
Does 2FA affect the bots?

### DISCUSSION THREAD

**Discussion by SiPhung953** [marioparaschiv/nitro-sniper#290] [Q&A]
So I have 2 alts account, one I have 2FA on and the other one off, when I run the bot I only see the account without 2FA in action, I have already checked to see if I pasted my tokens right.
So does 2FA blocks the bot/ have any effect against the bot?
I forgot to mention but, my main account also have 2FA, so I wanna ask if it blocks the bots from sending the main account Nitro
![image](https://user-images.githubusercontent.com/93171516/139429908-8bd6508e-0480-4f08-9751-621f08c78e6c.png)

**Comment by Exorcism0666** [comment]
My account also has 2FA but since I have my token it works correctly.
As long as you have your token with your main account, this will also work
